client/src/Overview/StyleSelector/StyleSelectorOverview.jsx

```diff
@@ -21,6 +21,7 @@ function StyleSelectorOverview ({setSelectedStyle, setOriginalPrice, setSalePric
     .then(res => {
       let styleGuide = res.data;
       setStyles(styleGuide.results)
+      console.log (styleGuide.results, 'this is the styleguide')
```

Comment:
don't forget to remove console logs for PRs

Reply:
roger that! will push another PR request as soon as I remove it.